python/graphstorm/inference/utils.py

```diff
@@ -0,0 +1,91 @@
+"""
+    Copyright 2025 Contributors
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Utility functions for inference
+"""
+
+import os
+import shutil
+import tarfile
+
+
+def wrap_model_artifacts(path_to_model, path_to_yaml, path_to_json, path_to_entry,
+                         output_package_name='model', output_path=None):
```

Comment:
Let's try to avoid `package` because usually refers to an abstract collection of classes and interfaces, similar to a python module, e.g. we have [Java packages](https://docs.oracle.com/javase/tutorial/java/concepts/package.html).

Tar files are [usually referred to](https://en.wikipedia.org/wiki/Tar_(computing)) as `archives` or `tarballs`, let's use that naming.

Reply:
renamed to `output_tarfile_name`